.github/workflows/core.yml

```diff
@@ -13,12 +13,12 @@ jobs:
       matrix:
         os: [ubuntu-latest]
         # CPython 3.11 is in quick-test
-        python-version: ['3.8', '3.9', '3.10', '3.12', pypy-3.7, pypy-3.8, pypy-3.10]
+        python-version: ['3.8', '3.9', '3.10', '3.12', pypy-3.8, pypy-3.10]
         run-tests-ext: [sh]
         include:
         # atleast one of each CPython/PyPy tests must be in windows
         - os: windows-latest
-          python-version: '3.7'
+          python-version: '3.8'
```

Comment:
@Grub4K Why was 3.12 added for both windows and linux? https://github.com/yt-dlp/yt-dlp/pull/8130

Reply:
Tests "broke" on Windows but not on Linux (See https://github.com/yt-dlp/yt-dlp/issues/8137). We wanted to test oldest and newest version more extensively, but we have a limited amount of runners per project, so didnt want to add even more. Feel free to suggest better combinations